Russian README and CONTRIBUTING

achumakov opened the pull request:
Made a translation of readme etc. files. I'm unsure where to put them so put into languages/ru_RU/ 
Build history left untranslated so makes no sense. 'English only' remarks added to relevant links.

cojennin commented:
I've got a branch open investigating [automatically generating Edit Flow documentation](https://github.com/Automattic/Edit-Flow/pull/353) with PHPDocs. My hope is to eventually put it up on codex.editflow.org, just need to work on getting the domain information figured out. Then we could work on having multilingual documentation like [WordPress core](https://codex.wordpress.org/Codex:Multilingual).

In the meantime, I think grouping it as `documentation/ru_RU/` would be great! Then we can group all the documentation related material in one place (and hopefully we eventually move it to codex.editflow.org). I don't know much about translating PHPDocs, but once I get that working it would be great if you have some time and would be interested to figure out how to do it!

PR is good to go once that change (grouping everything under `documentation`) is made!

achumakov commented:
As suggested, moved to documentation/ru_RU.
Also, replaced build history from Russian readme with just a reference for now - DRY ;)

Ready for merge.

cojennin commented:
Awesome, thanks for making those changes!
